### DISCUSSION TITLE
Name

### DISCUSSION THREAD

**Discussion by Secret-chest** [Secret-chest/scratch2python#23] [Meta]
Some users [pointed out](https://scratch.mit.edu/discuss/topic/583651/?page=1#post-6087511) that my project could be confused with [Scratch2Py](https://github.com/The-Cloud-Dev/scratch2py), which is for cloud variable management.

To solve this, you can suggest a new name here. It'd be better if it didn't contain "Scratch", "Python", "py", or anything similar, because if I were to change the name, I would consider Scratch2Python a temporary one and wait for a more interesting one.

**Comment by imdevarsh** [comment]
Meowsnake?

Meow - Scratch
Snake - Python

**Reply by Secret-chest** [reply]
Nice, but I'll wait for the others

**Reply by user444notfound** [reply]
I think meow2snake would be better than meowsnake.

**Comment by user444notfound** [comment]
Maybe cat2snake, but I think the best name is scratch2python .

**Reply by Secret-chest** [reply]
> Maybe cat2snake, but I think the best name is scratch2python .

But Scratch2Py was first.

I would like a more imaginative name.

**Comment by Secret-chest** [comment]
Pythonplayer?

**Reply by user444notfound** [reply]
I have a new idea! TheCatMeetsTheSnake @Secret-chest

**Reply by Secret-chest** [reply]
> I have a new idea! TheCatMeetsTheSnake @Secret-chest

Too long.

**Reply by imdevarsh** [reply]
Pyplayer

**Reply by Secret-chest** [reply]
> Pyplayer

I don't like how the letters are repeated, sorry

**Comment by AishikDasgupta** [comment]
How about, 
Cat2Python
Resembles kinda the old name, but is different...

**Comment by imdevarsh** [comment]
Meowconvert

**Comment by imdevarsh** [comment]
Pymeow

**Comment by Secret-chest** [comment]
I have an idea!

Catinterpreter

**Comment by imdevarsh** [comment]
Slithercat

**Reply by Secret-chest** [reply]
As of today, I think this name is the best.

**Comment by Secret-chest** [comment]
Cattranslator.

**Comment by Rudxain** [comment]
TigerVenom?

**Reply by Secret-chest** [reply]
I think I'd go for something a little cuter, but I like it. I may use that.

**Reply by Rudxain** [reply]
I agree, I also thought my suggestion sounded too violent, lol

**Reply by Secret-chest** [reply]
Oh, and pythons don't have venom. Maybe CompressedCat?

**Reply by Rudxain** [reply]
> pythons don't have venom.

True. I searched it on Google, but I couldn't think of a better replacement

> Maybe CompressedCat?

That gave me an idea: *e l o n g a t e d   cat*, lmao
